src/pycram/costmaps.py

```diff
@@ -80,16 +82,65 @@ def __init__(self, resolution: float,
         self.map: np.ndarray = map
         self.vis_ids: List[int] = []
 
+    # def visualize(self) -> None:
+    #     """
+    #     Visualizes a costmap in the World, the visualisation works by
+    #     subdividing the costmap in rectangles which are then visualized as world visual shapes.
+    #     """
+    #     if self.vis_ids:
+    #         return
+    #
+    #     # working on a copy of the costmap, since found rectangles are deleted
+    #     map = np.copy(self.map)
+    #     boxes = []
+    #     # Finding all rectangles in the costmap
+    #     for i in range(0, map.shape[0]):
```

Comment:
for keeping track of old code we got version control. Blocks of commented code is a code smell.